satpy/readers/avhrr_l1b_gaclac.py

```diff
@@ -57,7 +57,7 @@ class GACLACFile(BaseFileHandler):
     def __init__(self, filename, filename_info, filetype_info,
                  start_line=None, end_line=None, strip_invalid_coords=True,
                  interpolate_coords=True, adjust_clock_drift=True,
-                 tle_dir=None, tle_name=None, tle_thresh=7):
+                 tle_dir=None, tle_name=None, tle_thresh=7, calibration=None):
```

Comment:
I think in order to keep good compatibility with different versions of pygac and be able to provide more/different arguments in the future without the need to pr satpy, that we could replace the argument list with a **kwargs and refer to the docstring of the reader in pygac. What do you think?

Reply:
It would certainly be more flexible, but wouldn't we silently ignore typos then?

Reply:
Ah, no, because the pygac reader would complain about invalid kwargs then.

Reply:
Yes exactly